README.md

```diff
@@ -0,0 +1,123 @@
+# ActiveStorageBase64
+
+Gem used to add support for base64 images for Rails's ActiveStorage.
+
+## Getting Started
+
+In order to get the gem working on your project you just need to add the gem to your project like this:
+```ruby
+gem 'active-storage-base64'
+```
+
+### Prerequisites
+
+The only two prerequisites for using this gem are having Rails version 5.2.0 or higher installed on your project and having ActiveStorage setup properly (for more information on how to do this, check this [Active Storage Overview](https://edgeguides.rubyonrails.org/active_storage_overview.html))
+
+```ruby
+gem 'rails', '5.2.0'
+```
+
+### Installing
+
+In order to use the gem's functionality, you'll need to include the module into your ActiveRecord inheriting class.
+For example:
+```ruby
+class Test < ActiveRecord::Base
+    include ActiveStorageSupport::SupportForBase64
```

Comment:
Wrong indentation (4 spaces instead of 2). 
Also, why not recommending including the module in ApplicationRecord? So that every model could use our method out of the box?

Reply:
Sure thing! Will add it